src/rules/font-family-no-duplicate-names/README.md

```diff
@@ -0,0 +1,47 @@
+# font-family-no-duplicate-names
+
+Disallow duplicate font family names.
+
+```css
+a { font-family: "Times New Roman", serif; }
+/**               ↑                 ↑
+ *               These font family names */
+```
+
+This rule checks the `font` and `font-family` properties.
+
+This rule ignores `$sass`, `@less`, and `var(--custom-property)` variable syntaxes.
+
+**Caveat:** This rule does not currently understand escape sequences. If you want to use the font family name "Hawaii 5-0" you will need to wrap it in quotes, instead of escaping it as `Hawaii \35 -0` or `Hawaii\ 5-0`.
```

Comment:
One can say so. The main idea - Please use quotation marks around multi-word font names or around font names containing disallowed symbols. Rather than use escape sequences in unquoted font family names. Because this rule does not currently support escape sequences.

Here some quotes from w3.org site:

> Font family names other than generic families must either be given quoted as strings, or unquoted as a sequence of one or more identifiers. This means most punctuation characters and digits at the start of each token must be escaped in unquoted font family names.

> To avoid mistakes in escaping, it is recommended to quote font family names that contain white space, digits, or punctuation characters other than hyphens